web/satellite/tests/unit/store/projectMembers.spec.ts

```diff
@@ -102,7 +123,8 @@ describe('actions', () => {
 				id: '1'
 			}
 		};
-		addProjectMembersRequest = jest.fn().mockReturnValue({isSuccess: false});
+		jest.spyOn(api, 'addProjectMembersRequest').mockReturnValue({isSuccess: false});
```

Comment:
will new Response<nil>() will be better for understanding instead of {isSuccess: false}?